### ISSUE TITLE
Gutenberg create category default language not set for polylang

### ISSUE THREAD

**Issue by edemir206** [WordPress/gutenberg#13039]
**Describe the bug**

Polylang and Gutenberg compatibility issue, when creating category on gutenberg does not set default language for polylang.

**To Reproduce**
1. Install Wordpress 5.0.2 or latest Version
2. Install Polylang
3. In Gutenberg editor try adding a new category
4. Go to "Posts" > "Categories"
5. The category created within Gutenberg does not have the default language set.

When I follow those steps, I see...
The category default language comes undefined

I was expecting...
The category default language should come predefined

**Screenshots**
1) Gutenberg actions

![image](https://user-images.githubusercontent.com/1863099/50286264-07682b80-0446-11e9-8e6f-d70bdfa9cca0.png)

2) "Posts" > "Categories" page view, the default language should be set.

![image](https://user-images.githubusercontent.com/1863099/50286349-3da5ab00-0446-11e9-8a89-86f7b2ba4c99.png)

3) Polylang Lang View
![image](https://user-images.githubusercontent.com/1863099/50286512-ba388980-0446-11e9-9482-743e29edb593.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

I don't know if this issue is gutenberg related or polylang, so I opened bug tickets on both git repositories.

**Comment by swissspidy** [comment]
Have you seen https://polylang.pro/polylang-2-5-and-wordpress-5-0/ and https://github.com/polylang/polylang-gutenberg?

Have you contacted Polylang support already about this?

**Comment by edemir206** [comment]
> Have you seen https://polylang.pro/polylang-2-5-and-wordpress-5-0/ and https://github.com/polylang/polylang-gutenberg?
> 
> Have you contacted Polylang support already about this?

Thanks,

The link you sent explains this issue.

![image](https://user-images.githubusercontent.com/1863099/50288868-531ed300-044e-11e9-9007-7081c978d0a3.png)

The problem is, we have a large network of multisites, and some of them are having problems when you create the category within Gutenberg, they are not appearing on "Posts" > "Categories"  before you set "all posts and categories" to default language.

Disabling polylang fixes this problem for now, but creates an issue, we need to fix like 61 sites menus that have default language set because the just disappear.

What is your suggestion ? Should we disable Polylang or switch back to classic editor for now ?

**Comment by swissspidy** [comment]
> The link you sent explains this issue.

Correct. That link also explains that the issue is fixed in Polylang **Pro** but not Polylang. Which means they're probably working on fixing it there as well.

I see that you created https://github.com/polylang/polylang/issues/323 as well. Please continue discussing this with the Polylang team there.

In the meantime using the Classic Editor for certain posts can be an option. It's what I'm doing as well because Polylang and Gutenberg don't yet work together that well.
